kodi_voice/kodi.py

```diff
@@ -50,6 +50,14 @@ def sanitize_name(media_name, remove_between=False, normalize=True):
   name = name.strip()
   return name
 
+def sanitize_channel(name, lang='en'):
+  name = name.lower()
+  name = name.replace("+", " plus ")
+  name = re.sub(r"\sch\s", " channel ", name)
+  name = re.sub("^channel", "", name)
+  name = re.sub(r"(?<=\D)(?=\d)|(?<=\d)(?=\D)", " ", name)
+  name = words2digits(name, lang=lang)
```

Comment:
I think where you're getting confused is that sanitize_name() was for sanitizing the output from Kodi for the slots, primarily.  It's not intended to sanitize the user's input.

We handle input-mangling (that is, stuff from Alexa) in Kodi.matchHeard(), where it tries to massage the input to more closely match how it's stored in Kodi.